internal/log/log_unsupported.go

```diff
@@ -3,7 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-//go:build (!cgo && ((!darwin && !freebsd) || go1.26)) || datadog.no_waf
+//go:build (!linux && !darwin) || (!amd64 && !arm64) || go1.26 || datadog.no_waf || (!cgo && !appsec)
```

Comment:
TBH this could just be

```suggestion
//go:build !linux || !darwin || !amd64 || !arm64 || go1.26 || datadog.no_waf || !cgo || !appsec
```